{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/admin.py

```diff
@@ -1,19 +1,18 @@
 from django.conf import settings
 from django.contrib import admin
 from django.contrib.auth import admin as auth_admin
-from django.contrib.auth import decorators
-from django.contrib.auth import get_user_model
+from django.contrib.auth.decorators import login_required
 from django.utils.translation import gettext_lazy as _
 
 from {{ cookiecutter.project_slug }}.users.forms import UserAdminChangeForm
 from {{ cookiecutter.project_slug }}.users.forms import UserAdminCreationForm
+from {{ cookiecutter.project_slug }}.users.models import User
```

Comment:
Ruff doesn't like my import order for some reason and I can't figure out why

Reply:
TIL ruff wants two lines before class definitions but one line before statements